packages/blockprotocol/core.d.ts

```diff
@@ -49,6 +49,10 @@ export type BlockMetadata = {
    * A short description of the block, to help users understand its capabilities
```

Comment:
Might as well bump the version of this too, no? Could publish it immediately and save adding to the patch in HASH

Reply:
The current size of `blockprotocol` patch is quite large, which means that bumping `blockprotocol` in HASH will potentially require unrelated fixes in https://github.com/hashintel/hash/pull/603. I merged that PR to keep it focused. We can still bump `blockprotocol` package here, but upgrade a version in a separate HASH PR.